internal/pkg/template/templates/environment/partials/environment-manager-role.yml

```diff
@@ -19,14 +19,25 @@ EnvironmentManagerRole:
       PolicyDocument:
         Version: '2012-10-17'
         Statement:
-{{- if .ImportCertARNs}}
+{{- if .PublicImportedCertARNs}}
         - Sid: ImportedCertificates
           Effect: Allow
           Action: [
             acm:DescribeCertificate
           ]
           Resource:
-{{- range $arn := .ImportCertARNs}}
+{{- range $arn := .PublicImportedCertARNs}}
+          - "{{$arn}}"
+{{- end}}
+{{- end}}
+{{- if .PrivateImportedCertARNs}}
+        - Sid: ImportedCertificates
+          Effect: Allow
+          Action: [
+            acm:DescribeCertificate
+          ]
+          Resource:
+{{- range $arn := .PrivateImportedCertARNs}}
```

Comment:
folks should be able to import both public and private certs soon, so I think we can do:
```
{{ if or (.PublicImportedCertARNs .PrivateImportedCertARNs)}}
-  Sid: ImportedCertificates
   Effect: Allow
   Action: [
        acm:DescribeCertificate
   ]
   Resource:
{{- range $arn := .PublicImportedCertARNs}}
          - "{{$arn}}"
{{- end}}
{{- range $arn := .PrivateImportedCertARNs}}
          - "{{$arn}}"
{{- end}}
```